CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ We're so glad you're thinking about contributing to an 18F open source project!
 
 Before contributing, we encourage you to read our CONTRIBUTING policy (you are here), our LICENSE, and our README, all of which should be in this repository. If you have any questions, or want to read more about our underlying policies, you can consult the 18F Open Source Policy GitHub repository at https://github.com/18f/open-source-policy, or just shoot us an email/official government letterhead note to [18f@gsa.gov](mailto:18f@gsa.gov).
 
-## Public domain
+## Public Domain
```

Comment:
Actually, our [content guide advises that we use "sentence case"](https://pages.18f.gov/content-guide/capitalization/).